app/src/main/java/ai/elimu/appstore/service/GetAppListService.java

```diff
@@ -0,0 +1,21 @@
+package ai.elimu.appstore.service;
+
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * Retrofit integration with the REST API for downloading applications list
+ */
+public interface GetAppListService {
```

Comment:
Rename to `ApplicationService` to stay consistent with the naming conventions at https://github.com/elimu-ai/webapp/blob/master/REST_API_REFERENCE.md